.releaserc

```diff
@@ -0,0 +1,5 @@
+{
+    "branches": [
+        "master"
+    ]
```

Comment:
Could we also get [semantic-release/git](https://github.com/semantic-release/git) add here so it creates commits which update the version number after each release and pushes them? That would be nice :smiley: